src/dotnet/dotnet.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
i'll revert this file in the end - leaving as long as this is WIP since I'm gonna have to fight with VS now